src/clojure_tictactoe/core.clj

```diff
@@ -1,9 +1,12 @@
 (ns clojure-tictactoe.core
-  (:require [clojure-tictactoe.tictactoe :as ttt]))
+  (:require [clojure-tictactoe.cli.output.instructions-printer :as instructions-printer]
+            [clojure-tictactoe.cli.input.input-getter :as input-getter]
+            [clojure-tictactoe.cli.output.board-printer :as board-printer]
+            [clojure-tictactoe.game.game-play :as game-play]))
 
 (defn -main
   [& args]
-  (ttt/print-game-intro)
-  (ttt/continue-to-game)
-  (println (ttt/render-board (ttt/render-board-spaces)))
-  (ttt/game-loop))
+  (instructions-printer/print-game-intro)
+  (input-getter/continue-to-game)
+  (println (board-printer/render-board (board-printer/render-board-spaces)))
```

Comment:
This `main` function is starting look really good. This `println` though means that there's a direct dependency on the console here. Can you move that to a namespace that is responsible for UI?

Reply:
👍 This also inspired me to change render-board-spaces to render-empty-board and move it into the board namespace.